zsh/zshrc

```diff
@@ -106,3 +106,5 @@ export PATH="$HOME/.jbang/bin:$PATH"
 export PYENV_ROOT="$HOME/.pyenv"
 command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
 eval "$(pyenv init -)"
+
+load-nvm
```

Comment:
I think I removed this line for performance issues. With it, the zsh startup time is increased by 0.5s, you can check it with the following command: 

```
for i in {1..10}; do /usr/bin/time zsh -i -c exit; done
```

Right now the nvm is automatically loaded when you do a cd to a folder than contains a .nvmrc file.

Not sure if there is another alternative